addons/l10n_ec/data/l10n_ec.sri.payment.csv

```diff
@@ -1,9 +1,9 @@
-id,code,name
-P1,01,No use of the financial system
-P15,15,Offset of Debts
-P16,16,Debit Card
-P17,17,Electronic Cash
-P18,18,Prepaid Card
-P19,19,Credit Card
-P20,20,Others with use of the financial system
-P21,21,Endorsement of Securities
+id,code,name,active
+l10n_ec.P1,01,Sin utilización del sistema financiero,1
+l10n_ec.P15,15,Compensación de Deudas,1
+l10n_ec.P16,16,Tarjeta de Débito,1
+l10n_ec.P17,17,Dinero Electrónico,0
+l10n_ec.P18,18,Tarjeta de Prepago,0
+l10n_ec.P19,19,Tarjeta de Crédito,1
+l10n_ec.P20,20,Otros con utilización del sistema financiero,1
+l10n_ec.P21,21,Endoso de Títulos,0
```

Comment:
Hey @jco-odoo I think these terms should be in ENG and but with the name@es translations added.
```suggestion
id,code,name,active,name@es
l10n_ec.P1,01,No use of the financial system,1,Sin utilización del sistema financiero
l10n_ec.P15,15,Offset of Debts,1,Compensación de Deudas
l10n_ec.P16,16,Debit Card,1,Tarjeta de Débito
l10n_ec.P17,17,Electronic Cash,0,Dinero Electrónico
l10n_ec.P18,18,Prepaid Card,0,Tarjeta de Prepago
l10n_ec.P19,19,Credit Card,1,Tarjeta de Crédito
l10n_ec.P20,20,Others with use of the financial system,1,Otros con utilización del sistema financiero
l10n_ec.P21,21,Endorsement of Securities,0,Endoso de Títulos
```

Reply:
@vbe-odoo The name@es does not work if it is not a "template"

Reply:
We can do this later after the freeze, but then we will need to adapt the po files.

Reply:
Thanks for the clarification.
I can test, but it wouldn't be better if we add the translations now in the .po file already,
I can work the file, as it just a few translations, but the origin terms need to be in ENG.

Reply:
@vbe-odoo If you feel up to it...

Reply:
I can do it, as I already have the terms translated.
Can you help me change the origin terms to ENG?
Anyways the file should look something like the one attached here:
[PO FILE](https://drive.google.com/file/d/1l9FsbEuBw8s-WLUxvImafLp9Dl91SfG7/view?usp=drive_link)
**Disclosure**: in the PO file I changed the origin terms to ENG and then I added the ESP translations

I've tested the other changes and they LGTM.

Thanks!
cc' @fvz-odoo @ren-odoo

Reply:
Seems to me like this should already be translated, the strings en_US source strings are already present in `es.po`

Why are you changing it?